sceptre/cli/dump.py

```diff
@@ -48,8 +52,70 @@ def dump_config(ctx, path):
 
     output_format = "json" if context.output_format == "json" else "yaml"
 
+    def process_config(config):
+        stack_name = list(config.keys())[0]
+        file_path = Path(f".dump/{stack_name}/config.yaml")
+        write(config[stack_name], output_format, no_colour=True, file_path=file_path)
+
     if len(output) == 1:
-        write(output[0][stack.external_name], output_format)
+        process_config(output[0])
     else:
         for config in output:
-            write(config, output_format)
+            process_config(config)
+
+
+@dump_group.command(name="template")
+@click.argument("path")
+@click.option(
+    "-n",
+    "--no-placeholders",
+    is_flag=True,
+    help="If True, no placeholder values will be supplied for resolvers that cannot be resolved.",
+)
+@click.pass_context
+@catch_exceptions
+def dump_template(ctx, no_placeholders, path):
+    """
+    Prints the template used for stack in PATH.
+    \f
+
+    :param path: Path to execute the command on.
+    :type path: str
+    """
+    context = SceptreContext(
+        command_path=path,
+        command_params=ctx.params,
+        project_path=ctx.obj.get("project_path"),
+        user_variables=ctx.obj.get("user_variables"),
+        options=ctx.obj.get("options"),
+        output_format=ctx.obj.get("output_format"),
+        ignore_dependencies=ctx.obj.get("ignore_dependencies"),
+    )
+
+    plan = SceptrePlan(context)
+
+    execution_context = (
+        null_context() if no_placeholders else use_resolver_placeholders_on_error()
+    )
+    with execution_context:
+        responses = plan.dump_template()
+
+    output = []
+    for stack, template in responses.items():
+        if template is None:
+            logger.warning(f"{stack.external_name} does not exist")
+        else:
+            output.append({stack.external_name: template})
+
+    output_format = "json" if context.output_format == "json" else "yaml"
+
+    def process_template(template):
+        stack_name = list(template.keys())[0]
+        file_path = Path(f".dump/{stack_name}/template.yaml")
```

Comment:
@alexharv074 , to my point elsewhere, it seems you might not realize that logs are configured to go to stderr in Sceptre (it's actually the default for Python), so logs wouldn't interfere with using `>` because that sends stdout. They would only get sucked up if you used `$>`.

Reply:
Also, I might not be right about this, but I suspect this path isn't Windows-safe. You could instead do one of the following to guarantee it works with Windows:
```python
Path(".dump", stack_name, "template.yaml")
# or
Path(".dump") / stack_name / "template.yaml")

Reply:
Also, the template might be json, not yaml.

Reply:
Ok fixed fixed and fixed.